server/jetstream.go

```diff
@@ -49,10 +49,11 @@ type JetStreamAccountLimits struct {
 
 // JetStreamAccountStats returns current statistics about the account's JetStream usage.
 type JetStreamAccountStats struct {
-	Memory  uint64                 `json:"memory"`
-	Store   uint64                 `json:"storage"`
-	Streams int                    `json:"streams"`
-	Limits  JetStreamAccountLimits `json:"limits"`
+	Memory    uint64                 `json:"memory"`
+	Store     uint64                 `json:"storage"`
+	Streams   int                    `json:"streams"`
+	Consumers int                    `json:"consumers"`
```

Comment:
Matters more about them connected to a stream no?

Reply:
Hmm not sure I follow what you mean. Isn’t the limit about how many consumers are defined in total? In that case what I wrote seems to report on that usage

But maybe I miss understand the meaning of the JS limit. Will check again.

Reply:
consumer limits I thought was per stream. Might have a global one too.

Reply:
So the account consumers limit is the maximum amount of consumers any one stream can have rather than max consumers per account, that seems weird but ok, will close this PR

https://github.com/nats-io/nats-server/blob/1c5b8a9097160b15a9944c11cc4e71643cde170c/server/stream.go#L265